package/lib/Markdown.astro

```diff
@@ -14,7 +16,7 @@ if (!input) {
 }
 // @ts-ignore
 const components = createComponentProxy($$result, Astro.props.components);
-const content = await markdown(input, { sanitize: Astro.props.sanitize, components });
+const content = await markdown(input, { components }, Astro.props.markedExt);
```

Comment:
We still accept `sanitize` as a prop but it's silently ignored here. That feels weird!

I'd also prefer if the `markdown` function mirrored the `Props` interface as much as possible and did not introduce a third unlabeled argument. Typically, when the final argument is an object, that's meant to accept any options necessary for the API instead of adding new arguments. That's a bit of an implicit API design constraint, but it tends to aid in maintainability and refactorability.